CONTRIBUTING.md

```diff
@@ -21,18 +21,166 @@ To get an overview of the project, read the [README](README.md). Here are some r
+
 #### Adding images (in the right folder, code to do so)
 
+The visual editor also simplifies inserting images and customizing their display. You can paste an image directly from your clipboard into the visual editor, which will save a copy of the image in your project directory and link to it. Alternatively, you can use the Insert > Figure / Image menu to browse for the image you want to insert or paste its URL. This menu also allows you to resize the image, add captions, alternative text, and links.
+
+The visual editor offers many additional features that become evident as you gain more experience with it.
+
+Most importantly, while the visual editor displays your content with formatting, it saves everything in plain Markdown. You can easily switch between the visual and source editors to view and edit your content in either format.
+
 #### Adding links (within page, within site, external to site)\]
 
+In Quarto, adding links is straightforward using Markdown.
+
+In Markdown, you can add links using the following syntax:
+
+ 1. Inline Links:
+    [Link Text] ``(https://example.com)``
+
+ 2. Reference Links:
+    [Link Text][1]
+
+   `` [1]: https://example.com``
+    
+In Quarto, you can also include links within various contexts, such as code blocks or embedded within other content structures, depending on the complexity of your document. However, the basic Markdown link syntax will generally cover most use cases.
+ 
+ ![Example for adding links](images/adding_links_example.png){width=300}
+ 
 #### Adding videos / other content
+ 
+ In Quarto, you can add various types of content, including videos, to your documents. Here's how you can include videos and other content in a Quarto document when working with R Markdown.
+
+##### **Adding Videos**
+You can embed videos in Quarto documents using HTML, Markdown, or specialised Quarto syntax. Here are some methods:
+
+1. Using a shortcode:
+``` {.markdown shortcodes="false"} 
+{{< video url >}}
+``` 
+  - For example, here we embed a YouTube video:
+ 
+``` {.markdown shortcodes="false"} 
+{{< video https://www.youtube.com/embed/wo9vZccmqwc >}} 
+``` 
+{{< video https://www.youtube.com/embed/wo9vZccmqwc >}} 
+
+2. Embedding videos in other formats:
+  - In HTML formats the video will be embedded within the document.
+  - For other formats, a simple link to the video will be rendered.
+
+##### **Adding Other Content**
+ - **Audio Files**:
+ You can embed audio files using the HTML <audio> tag.
+ - **Interactive Content with HTML Widgets**:
+ Quarto supports various HTML widgets that can add interactive content to your document. 
+
+##### Tips 
+- **Relative Paths**:
+Use relative paths for local files to ensure your document remains portable. 
```

Comment:
Please could you find some guidance on relative paths and add a link here to it. Plenty of analysts will be unsure of what we mean by relative paths,